sftp/client.go

```diff
@@ -81,22 +81,37 @@ func NewSSHClient(config *SSHConfig) (*ssh.Client, error) {
 
 type clientImpl struct {
 	client *sftp.Client
+	config *SSHConfig
 }
 
 type Client interface {
```

Comment:
This is not needed as exported anymore, we can unexport it and use it as a local interface only.

Reply:
unexported